### ISSUE TITLE
Reference number on job poster unclear

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#58]
There is currently a reference number in the top right of the job poster, but it's really unclear what it's supposed to mean when going through the poster with a Screen Reader. I think we should probably add the words "reference number" or similar. At very least this text should be added for the screen reader.
